packages/dds/map/package.json

```diff
@@ -71,15 +71,14 @@
 		"build:test:esm": "tsc --project ./src/test/tsconfig.json",
 		"check:are-the-types-wrong": "attw --pack . --entrypoints .",
 		"check:biome": "biome check .",
-		"check:prettier": "prettier --check . --cache --ignore-path ../../../.prettierignore",
+		"check:format": "fluid-build --task check:format .",
```

Comment:
What's the difference between check:biome/check:format (and format/format:biome)?  Is one of the pair "correct" to call?  Do we need both?

I see you talk about these in the PR description but it doesn't quite seem to match up with what I see here?